packages/client/src/utils/validate.ts

```diff
@@ -293,11 +293,9 @@ export const isValidChildBirthDate: Validation = (value: IFormFieldValue) => {
 }
 
 export const isValidParentsBirthDate =
-  (minAgeGap: number, isAge?: boolean): Validation =>
```

Comment:
Do we not need this anymore?

Reply:
This was added by me. I just revert it back.

Reply:
I thought it will not be needed. My bad. Revert it back.